tembo-operator/tests/integration_tests.rs

```diff
@@ -2037,12 +2080,121 @@ mod test {
             }
             i += 1;
         }
-        // Get the ingress route tcp
         let ing_route_tcp = ingress_route_tcp_api.get(&ing_route_tcp_name).await;
-        // Should be deleted
         assert!(ing_route_tcp.is_err());
 
-        // Cleanup CoreDB resource
+        // Enable Dedicated Networking Test
+        let context = state.create_context(client.clone());
+
+        let coredb_json = serde_json::json!({
+            "apiVersion": API_VERSION,
+            "kind": kind,
+            "metadata": {
+                "name": name
+            },
+            "spec": {
+                "dedicatedNetworking": {
+                    "enabled": true,
+                    "includeStandby": true,
+                    "public": true,
+                    "serviceType": "LoadBalancer"
+                }
+            }
+        });
+        let params = PatchParams::apply("functional-test-dedicated-networking");
+        let patch = Patch::Apply(&coredb_json);
+        let _coredb_resource = coredbs.patch(name, &params, &patch).await.unwrap();
+
+        tokio::time::sleep(Duration::from_secs(10)).await;
```

Comment:
this looks like a sleep in order to wait for something, but instead it should have a wait retry loop to prevent test error if cluster is slow or busy